documents/src/pages/elements/tree-select.md

```diff
@@ -132,7 +132,7 @@ el.data = [{
 ```
 ::
 
-The `data` property of the `ef-tree-select` uses the [TreeSelectData](https://github.com/Refinitiv/refinitiv-ui/blob/develop/packages/elements/src/tree-select/helpers/types.ts) interface for its data items.
+The `data` property of the `ef-tree-elect` uses the [TreeSelectData](https://github.com/Refinitiv/refinitiv-ui/blob/v7/packages/elements/src/tree-select/helpers/types.ts) type for its data items. Each of this item defaults to type [TreeSelectDataItem](https://github.com/Refinitiv/refinitiv-ui/blob/v7/packages/elements/src/tree-select/helpers/types.ts). It could also be anything extended from [DataItem](./custom-components/utils/data-management#data-item).
```

Comment:
```suggestion
The `data` property of the `ef-tree-elect` uses the [TreeSelectData](https://github.com/Refinitiv/refinitiv-ui/blob/v6/packages/elements/src/tree-select/helpers/types.ts) type for its data items. Each of this item defaults to type [TreeSelectDataItem](https://github.com/Refinitiv/refinitiv-ui/blob/v6/packages/elements/src/tree-select/helpers/types.ts). It could also be anything extended from [DataItem](./custom-components/utils/data-management#data-item).
```